### ISSUE TITLE
NVDA cannot detect the tag picker (combined box compoent) in browser mode

### ISSUE THREAD

**Issue by crazyLayne** [nvaccess/nvda#11461]
### Steps to reproduce:
1. Open Chrome
2. Browse to https://codepen.io/layne_wright/pen/MWygXVQ?__cf_chl_jschl_tk__=b2cf5e7d7cc0ca0164699757ea3ea84139928237-1596501668-0-AfFH_uqNqn84Ea0iXCA7U0Bw-W9ddkHxitj8NtT2fE5Q4Vd79KZmyGxQiDczESdfYjkU1MYgU4Nje6vFho7Rangs47B3zSnQv4vZlt1VNQ0qWvhc6ihBompeST9Juu5WP3yplHGBVOrUZ0wRjV50IONOLh21XlSzdf6s4eugKSJapWwbXvVexi0PK77ZvJ8oUMBdFIdfqtARH5zG0jV31i3oRODVKkwGH-gdhdjekCRMtKHk-OBAP0OuBPOKN3h-aJFWpHFoiDBzH2WdEZXzZTgR26BUXJfBNsaWE5gTbgtGsBcHgX4-I8E__snWDkbQUk6rI2hjaFjLuNcx-QH7d7ST0dRgrZ7C4VmW3Liv74Os
3. Open NVDA to visit the string "Start of tag picker"
4. Press Down key

### Actual behavior:
It skips the tag picker and directly go to the string "End of tag picker"

### Expected behavior:
It should announce the tag picker

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2020.2

#### Windows version:
Win 10 enterprise
version: 10.0.18363 Build 18363

#### Name and version of other software in use when reproducing the issue:
version 84.0.4147.105 (Official Build) (64-bit)

#### Other information about your system:
Running a Win 10 on a laptop

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by Adriani90** [comment]
I think this is a Chrome issue, In Firefox it works as expected.
cc: @ObjectInSpace, @aleventhal.

**Comment by ObjectInSpace** [comment]
Filed https://crbug.com/1116148. Thanks much!

**Comment by Adriani90** [comment]
@crazyLayne are you still having this issue with NVDA 2024.2 Stable or 2024.3 Beta? The minimal test case doesn't work anymore.

**Comment by Adriani90** [comment]
We didn't hear any updates from the initial author upon request since many years, closing as abandoned. if you are still having this issue, please comment and we can reopen.
